### PR TITLE
Add custom parameter

### PR DESCRIPTION
**Pull request by wontem**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5522,7 +5522,7 @@
         },
         "readable-stream": {
           "version": "1.0.34",
-          "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-1.0.34.tgz",
+          "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-1.0.34.tgz",
```
Comment (dnechay) [inline package-lock.json:5525]: Is there a reason for it to be http and not https?

Comment (wontem) [reply package-lock.json:5525]: Not sure, that's how npm generated the file

Comment (wontem) [reply package-lock.json:5525]: Interesting, why for this package only

Comment (dnechay) [reply package-lock.json:5525]: That's a riddle
I presume it should be `https`, version bump shouldn't update any URLs

Comment (wontem) [reply package-lock.json:5525]: Version bump regenerates the lock file anyway.

Here is what I found
https://npm.community/t/npm-install-downgrading-resolved-packages-from-https-to-http-registry-in-package-lock-json/1818

Comment (wontem) [reply package-lock.json:5525]: Will try to remove `node_modules` and run install one more time

Comment (dnechay) [reply package-lock.json:5525]: If it doesn't help we might need to bring it back to `https` manually

Comment (wontem) [reply package-lock.json:5525]: Don't think it has any sense, lock files shouldn't be touched manually. I think that the problem is, I'm using `node@16` that uses `npm` that generates lock version `2`, so I installed deps, then found that the lock file is completely changed, reverted it, switched to `node@14`, and changed `package.json` version. I think these steps affected the resulting lock file after version change.

Ok, removed `node_modules`, ran `npm install`, and the problem went away

**Review by dnechay — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (wontem) [reply package-lock.json]: Looks like `npm@7` resolves the package verion to `http` while `npm@6` doesn't

Comment (dnechay) [reply package-lock.json]: @wontem will take it into account when will be switching versions, thank you for investigating this

**Pull request merged**